html/interaction/focus/sequential-focus-navigation-and-the-tabindex-attribute/focus-tabindex-order-shadow-zero-slot-one.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML Test: focus - the sequential focus navigation order with shadow dom and all tabindex=0, except slot (tabindex=1)</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<body>
+<div id="aboveHost">aboveHost</div>
+<div id="host">
+  <div id="slotted">slotted</div>
+</div>
+<div id="belowHost">belowHost</div>
+</body>
+
+<script>
+const shadowRoot = host.attachShadow({ mode: "open" });
+const aboveSlot = document.createElement("div");
+aboveSlot.innerText = "aboveSlot";
+const slot = document.createElement("slot");
+const belowSlot = document.createElement("div");
+belowSlot.innerText = "belowSlot";
+shadowRoot.appendChild(aboveSlot);
+shadowRoot.appendChild(slot);
+shadowRoot.appendChild(belowSlot);
+
+// Structure:
+// <div #aboveHost tabindex=0>
+// <div #host tabindex=0>
+//    #shadowRoot
+//      <div #aboveSlot tabindex=0>
+//      <slot #slot tabindex=1>
+//        (slotted) <div #slotted tabindex=0>
+//      <div #belowSlot tabindex=0>
+// <div #belowHost tabindex=0>
+
+const elementsInFlatTreeOrder = [aboveHost, host, aboveSlot, slot, slotted, belowSlot, belowHost];
+
+function setAllTabIndex(value) {
+  for (let el of elementsInFlatTreeOrder) {
+    if (el == slot)
+      el.tabIndex = 1;
+    else
+      el.tabIndex = 0;
+  }
+}
+
+function navigateFocusForward() {
```

Comment:
Can you just use the shared one and then say `const focused = shadowRoot.activeElement || document.activeElement` afterward?

Reply:
Done.